### PR TITLE
Updating dict.get default = 1

### PR DESCRIPTION
**Pull request by Qudirah**
fixes for #6443

### CODE REVIEW

**Comment by Qudirah** [comment]
@rossbar Please I do not understand why it keeps canceling. There isn't a particular error. I need help :(

File: networkx/utils/heaps.py [hunk 1]
```diff
@@ -64,7 +64,7 @@ def pop(self):
         """
         raise NotImplementedError
 
-    def get(self, key, default=None):
+    def get(self, key, default=1):
```
Comment (MridulS) [inline networkx/utils/heaps.py:67]: Hmm, why this change? What is this pull request supposed to do?

**Comment by MridulS** [comment]
Also before sending a PR please do run the tests locally https://networkx.org/documentation/latest/developer/contribute.html#testing, this will catch all the issues before creating a pull request :)

File: networkx/utils/heaps.py (hunk 1 above)
Comment (Qudirah) [reply networkx/utils/heaps.py:67]: My bad. I was getting frustrated because the test kept failing and I didn't link the issue. It's #6443. Have I implemented it wrongly?

**Comment by Qudirah** [comment]
> Also before sending a PR please do run the tests locally https://networkx.org/documentation/latest/developer/contribute.html#testing, this will catch all the issues before creating a pull request :)

Got it. Thanks.

File: networkx/utils/heaps.py (hunk 1 above)
Comment (MridulS) [reply networkx/utils/heaps.py:67]: Yes, this bit of code is for heaps. I don't think this has anything to do with GEXF export.

What @dschult meant in https://github.com/networkx/networkx/pull/6484#issuecomment-1465016028 was to look at dict.get in python https://docs.python.org/3/library/stdtypes.html#dict.get

An example:
``` python
>>> d = {1: 1, 2: 2}
>>> d.get(1)
1
>>> d.get(3)  # by default it gives None
>>> d.get(3, 'something') # you can change the default by providing another value
'something'
```

Comment (Qudirah) [reply networkx/utils/heaps.py:67]: smh. My bad. Thank you for explaining. will correct it now.

**Comment by Qudirah** [comment]
@MridulS Please check and merge if correct :D
